Mklinks subprocess -- rcsdiff -c -rR4-Vanilla -rR4-Mvex-r1 server/ddx/cfb/RCS/cfbgetsp.c,v 
*** 1.3.1.1	1991/09/22 02:30:18
--- 1.3.1.2	1991/09/22 18:30:18
***************
*** 68,74 ****
  	case 1:
  	    mfbGetSpans(pDrawable, wMax, ppt, pwidth, nspans, pdstStart);
  	    return;
! 	case 8:
  	    break;
  	default:
  	    FatalError("cfbGetSpans: invalid depth\n");
--- 68,74 ----
  	case 1:
  	    mfbGetSpans(pDrawable, wMax, ppt, pwidth, nspans, pdstStart);
  	    return;
! 	case PSZ:
  	    break;
  	default:
  	    FatalError("cfbGetSpans: invalid depth\n");
